syft/frameworks/crypten/__init__.py

```diff
@@ -1,58 +1,29 @@
-import torch
+import crypten
+import crypten.communicator as comm
 import syft
-
 from syft.frameworks.crypten.context import run_party
 
-import crypten.communicator as comm
-import crypten
 
+def load(tag: str, src: int = 0, **kwargs):
+    """Load an object tagged with 'tag' located at the syft worker running
+    party with rank 'src'.
 
-def load(tag: str, src: int):
-    if src == comm.get().get_rank():
-        # Means the data is on one of our local workers
+    Args:
+        tag: tag of the object to be looked up.
+        src: rank of the party that should attempt to fetch the object.
+    """
 
-        worker = syft.local_worker.get_worker_from_rank(src)
-        results = worker.search(tag)
+    if src == comm.get().get_rank():
+        results = syft.local_worker.search(tag)
```

Comment:
Q: Is this working with virtual workers? I remember that I added that ```get_worker_from_rank``` because of the VirtualWorkers. In case we send data to Alice and Bob, the local worker will not have that specific tag, but the VirtualWorker)

Reply:
Oh yeah, I think it should be 
```suggestion
    if src == comm.get().get_rank():
        worker = syft.local_worker.get_worker_from_rank(src)
        results = worker.search(tag)
```
right?

Reply:
Yep